quattor/types/metadata.pan

```diff
@@ -4,6 +4,10 @@ type structure_branch = {
     "author" ? string
     "name" : string
     "type" : string
+    @{a unique id, e.g. hash of last git commit}
+    "id" ? string
```

Comment:
id could be a bit mis-leading/ambiguous. What's the name of the pan command option that would fill this in?  How about "profile version" since that's really what this is.